code/src/cljs/sixsq/nuvla/ui/deployment_dialog/views_infra_services.cljs

```diff
@@ -79,7 +79,8 @@
 
 
 (defn creds-list
-  []
+  ;; TODO: generate creds list taking subtype into the account.
```

Comment:
TODO to be remove?

Reply:
Yes, I will remove this, as apparently the list of the creds already pre-filtered for the type of the IS the deployment will be started on.

Reply:
Done in 25d19e9